Corrige a validação de tamanho de args

formoso opened the pull request:
Este commit corrige a verificação do número de argumentos que são passados junto com o executavel

rformoso committed:
Corrige a validação de tamanho de args

Este commit corrige a verificação do número de argumentos que são passados junto com o executavel

EmersonFrancisco commented on main.go:
```diff
@@ -8,7 +8,7 @@ import (
 
 func main() {
-	if len(os.Args) < 1 {
+	if len(os.Args) <= 1 {
```
seria melhor colocar <2 por ser mais facil de ler
```suggestion
	if len(os.Args) <2  {
```

formoso replied:
Gostei

EmersonFrancisco reviewed (changes requested):
apenas uma dica

rformoso committed:
mend

rformoso committed:
Corrige a validação de tamanho de args

Este commit corrige a verificação do número de argumentos que são passados junto com o executavel

EmersonFrancisco reviewed (approved):
Okay
